examples/mean/tests/test_mean.py

```diff
@@ -21,72 +17,80 @@ class StreamBusMonitor(BusMonitor):
     _signals = ["valid", "data"]
 
     @cocotb.coroutine
-    def _monitor_recv(self):
+    async def _monitor_recv(self):
         """Watch the pins and reconstruct transactions."""
 
         while True:
-            yield RisingEdge(self.clock)
-            yield ReadOnly()
+            await RisingEdge(self.clock)
+            await ReadOnly()
             if self.bus.valid.value:
                 self._recv(int(self.bus.data.value))
 
 
 @cocotb.coroutine
-def clock_gen(signal, period=10, units='ns'):
+async def clock_gen(signal, period=10, units='ns'):
```

Comment:
This entire thing should be replaced with `Clock`.

Reply:
Done